OutOfSchool/OutOfSchool.WebApi/Services/MinistryAdminService.cs

```diff
@@ -0,0 +1,165 @@
+﻿using AutoMapper;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Options;
+
+using Newtonsoft.Json;
+using OutOfSchool.Common.Models;
+using OutOfSchool.WebApi.Enums;
+using OutOfSchool.WebApi.Models;
+
+namespace OutOfSchool.WebApi.Services;
+
+public class MinistryAdminService : CommunicationService, IMinistryAdminService
+{
+    private readonly IdentityServerConfig identityServerConfig;
+    private readonly IEntityRepository<User> userRepository;
+    private readonly IInstitutionAdminRepository institutionAdminRepository;
+    private readonly ILogger<MinistryAdminService> logger;
+    private readonly IMapper mapper;
+    private readonly ResponseDto responseDto;
+
+    public MinistryAdminService(
+        IHttpClientFactory httpClientFactory,
+        IOptions<IdentityServerConfig> identityServerConfig,
+        IOptions<CommunicationConfig> communicationConfig,
+        IInstitutionAdminRepository institutionAdminRepository,
+        IEntityRepository<User> userRepository,
+        IMapper mapper,
+        ILogger<MinistryAdminService> logger)
+        : base(httpClientFactory, communicationConfig.Value)
+    {
+        this.identityServerConfig = identityServerConfig.Value;
+        this.institutionAdminRepository = institutionAdminRepository;
+        this.userRepository = userRepository;
+        this.mapper = mapper;
+        this.logger = logger;
+        responseDto = new ResponseDto();
```

Comment:
Please doing nullable check the input parameters

Reply:
fixed

Reply:
please see the commit [fix Null Check Arguments](https://github.com/ita-social-projects/OoS-Backend/pull/747/commits/e533650346ecb592426fdd6991df15fdb1b5f2b8)